tajo-common/src/main/java/org/apache/tajo/SessionVars.java

```diff
@@ -126,6 +126,9 @@
   NULL_CHAR(ConfVars.$TEXT_NULL, "null char of text file output", DEFAULT),
   CODEGEN(ConfVars.$CODEGEN, "Runtime code generation enabled (experiment)", DEFAULT),
 
+  TABLE_PARTITION_KEPT_EXISTING_DATA_ENABLED(ConfVars.$TABLE_PARTITION_KEPT_EXISTING_DATA_ENABLED,
+    "When inserting empty data into a partitioned table, set to keep existing data.", DEFAULT),
```

Comment:
It would be better if the description is 'If True, a partitioned table is overwritten even if a sub query leads to no result. Otherwise, the table data will be kept if there is no result.'.